pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/util/EasyPostUtilTest.java

```diff
@@ -0,0 +1,30 @@
+package org.broadinstitute.dsm.util;
+
+import com.easypost.EasyPost;
+import com.easypost.exception.EasyPostException;
+import com.easypost.model.Address;
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.ddp.util.ConfigManager;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class EasyPostUtilTest {
+
+    private EasyPostUtil easyPostUtil =  EasyPostUtil.initializeEasyPostFromTypeSafeConfig();
+
+    @Test
+    public void testNullAddress2() {
+        if (StringUtils.isBlank(EasyPost.apiKey)) {
+            EasyPost.apiKey = ConfigManager.getInstance().getConfig().getString("easyPostApiKey");
+        }
```

Comment:
It feels like we are avoiding rather than testing a basic service capability, although perhaps I am not understanding this correctly. 
Perhaps the `initializeEasyPostFromTypeSafeConfig` call should go here, and if it fails the test fails? This code seems to be circumventing the behavior of the module under test, rather than testing this aspect of its behavior.

Reply:
Actually I meant to remove L17-19.  Will do that.  I had factored out the common code to find a usable `apiKey`.